src/voice-search/__tests__/voice-search.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { createRenderer } from '../../../helpers/test-renderer';
+import { NgAisVoiceSearch } from '../voice-search';
+
+const defaultState = {
+  isBrowserSupported: true,
+  voiceListeningState: {},
+};
+
+const render = createRenderer({
+  defaultState,
+  template: '<ais-voice-search></ais-voice-search>',
+  TestedWidget: NgAisVoiceSearch,
+});
+
+describe('VoiceSearch', () => {
+  describe('button', () => {
+    it('calls toggleListening when button is clicked', () => {
+      const toggleListening = jest.fn();
+      const fixture = render({
+        toggleListening,
+      });
+      const button = fixture.debugElement.nativeElement.querySelector('button');
+      button.click();
+      expect(toggleListening).toHaveBeenCalledTimes(1);
+    });
```

Comment:
We can add another click and check that `toggleListening` was called a second time to check that the button is still clickable when listening.

Reply:
[2fad3d1](https://github.com/algolia/angular-instantsearch/pull/530/commits/2fad3d12057c0fc389f23ddd144547932c1ef0e9)